kill-all.sh

```diff
@@ -1,4 +1,38 @@
 #!/usr/bin/env bash
 
-sudo docker kill agraf aprom
-sudo docker rm agraf aprom
+usage="$(basename "$0") [-h] [-g grafana port ] [ -p prometheus port ] -- kills existing Grafana and Prometheus Docker instances at given ports"
+
+while getopts ':hg:p:' option; do
+  case "$option" in
+    h) echo "$usage"
+       exit
+       ;;
+    g) GRAFANA_PORT=$OPTARG
+       ;;
+    p) PROMETHEUS_PORT=$OPTARG
+       ;;
+    :) printf "missing argument for -%s\n" "$OPTARG" >&2
+       echo "$usage" >&2
+       exit 1
+       ;;
+   \?) printf "illegal option: -%s\n" "$OPTARG" >&2
+       echo "$usage" >&2
+       exit 1
+       ;;
+  esac
+done
+
+if [ -z $GRAFANA_PORT ]; then
+    GRAFANA_NAME=agraf
+else
+    GRAFANA_NAME=agraf-$GRAFANA_PORT
+fi
+if [ -z $PROMETHEUS_PORT ]; then
+    PROMETHEUS_NAME=aprom
+else
+    PROMETHEUS_NAME=aprom-$PROMETHEUS_PORT
+fi
+
+
+sudo docker kill $GRAFANA_NAME $PROMETHEUS_NAME
+sudo docker rm $GRAFANA_NAME $PROMETHEUS_NAME
```

Comment:
@glommer the kill script become pretty big
Would it help to fold it as an option under the start script?